packages/circuit-ui/components/Select/Select.stories.tsx

```diff
@@ -48,7 +48,7 @@ const baseArgs = {
   noMargin: true,
 };
 
-const flagIconMap: { [key: string]: FC<{ className?: string }> } = {
+const flagIconMap: { [key: string]: FC<IconProps<'16'>> } = {
```

Comment:
Let's remove the `FC` here as well as it incorrectly adds the `children` prop to the icons.

Reply:
No, `FC` doesn't add `children` anymore ([v18 TS change](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-typescript-definitions)). Icons are also functional components, so we can't type them as `IconProps`. Any other suggestions?

Reply:
Oh, that's cool!